build-aux/m4/ax_boost_base.m4

```diff
@@ -147,6 +147,7 @@ if test "x$want_boost" = "xyes"; then
     if test "$ac_boost_lib_path" != ""; then
        BOOST_LDFLAGS="-L$ac_boost_lib_path"
     fi
+    BITCOIN_SYSTEM_INCLUDE([BOOST_CPPFLAGS], [$BOOST_CPPFLAGS])
```

Comment:
These changes should be done outside of ax_boost_base.m4. This is an upstream file.

Reply:
Fixed